vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericSetVariableSteps.java

```diff
@@ -230,4 +231,61 @@ private void saveAttributeValueOfElement(Optional<WebElement> element, String at
             return value;
         }).ifPresent(value -> variableContext.putVariable(scopes, variableName, value));
     }
+
+    /**
+     * Saves the information about the size of an element and its position relative to the viewport.
+     *
+     * The following values can be accessed by using "dot" notation on the variable name:
+     * <table>
+     * <caption>A table of rect attributes</caption>
+     * <thead>
+     * <tr>
+     *     <th><b>attribute</b></th>
+     *     <th><b>description</b></th>
+     * </tr>
+     * </thead>
+     * <tbody>
+     * <tr>
+     *     <td>x</td>
+     *     <td>the x coordinate</td>
+     * </tr>
+     * <tr>
+     *     <td>y</td>
+     *     <td>the y coordinate</td>
+     * </tr>
+     * <tr>
+     *     <td>height</td>
+     *     <td>the height of the element</td>
+     * </tr>
+     * <tr>
+     *     <td>width</td>
+     *     <td>the width of the element</td>
+     * </tr>
+     * </tbody>
+     * </table>
+     *
+     * @param locator        The locator to find an element
+     * @param scopes         The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     *                       <i>Available scopes:</i>
+     *                       <ul>
+     *                       <li><b>STEP</b> - the variable will be available only within the step,
+     *                       <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     *                       <li><b>STORY</b> - the variable will be available within the whole story,
+     *                       <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     *                       </ul>
+     * @param variableName   The name under which the rect should be saved
+     */
+    @When("I save rect of element located by `$locator` to $scopes variable `$variableName`")
+    public void saveElementRect(Locator locator, Set<VariableScope> scopes, String variableName)
+    {
+        baseValidations.assertElementExists("The element to get size and position", locator)
+                       .map(WebElement::getRect)
+                       .map(rect -> Map.of(
+                           "x", rect.getX(),
+                           "y", rect.getY(),
+                           "height", rect.getHeight(),
+                           "width", rect.getWidth())
+                       )
+                       .ifPresent(rect -> variableContext.putVariable(scopes, variableName, rect));
```

Comment:
what is logged here?

Reply:
<img width="769" alt="image" src="https://github.com/vividus-framework/vividus/assets/30923336/d66ba134-2d20-4581-8d1c-b2db2172814c">